<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uploadClass</class>
 <widget class="QMainWindow" name="uploadClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>upload</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QProgressBar" name="upload_progressBar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>561</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>上传进度:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="onepid_lineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>一级目录pid</string>
    </property>
   </widget>
   <widget class="QLabel" name="dir_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;上传根路径：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="selectdir_lineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>421</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="upload_pushButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>上传</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="upload.qrc"/>
 </resources>
 <connections/>
</ui>
